internal/pkg/cli/svc_package.go

```diff
@@ -123,14 +124,71 @@ func newPackageSvcOpts(vars packageSvcVars) (*packageSvcOpts, error) {
 		var serializer stackSerializer
 		switch t := mft.(type) {
 		case *manifest.LoadBalancedWebService:
-			var opts []stack.LoadBalancedWebServiceOption
-			if app.RequiresDNSDelegation() {
-				if err := validateLBSvcAliasAndAppVersion(aws.StringValue(t.Name), t.Alias, app, env.Name, appVersionGetter); err != nil {
+			if err := validateLBWSRuntime(app, env.Name, t, appVersionGetter); err != nil {
```

Comment:
@Lou1415926 @iamhopaul123 I'd love for us to propose a new package structure with the buildspec cleanup work so that these commands become lean.

It feels like most of the logic in these `case` blocks should move to the `deploy/cloudformation` pkg